03-task-manager/answers/routes/tasks.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const router = express.Router();
+
+const {
+	getAllTasks,
+	createTask,
+	getTask,
+	updateTask,
+	deleteTask,
+} = require('../controllers/tasks');
+
+router.route('/').get(getAllTasks).post(createTask);
+router.route('/:id').get(getTask).patch(updateTask).delete(deleteTask);
+// router.router('/:id').get(getTask).patch(updateTask).delete(deleteTask);
+
+router.route('/').get(getAllTasks);
```

Comment:
These can be removed; the comment is redundant; and the `route('/').get(getAllTasks)` has already been defined on line 12